scripts/migrate_locales.js

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env node
+/**
+ * A script to help integrators to migrate
+ * between different versions of Web SDK locale system.
+ *
+ * For more info, run:
+ *    $ migrate_locales --help
+ */
+
+'use strict' // eslint-disable-line strict
+
+const fs = require('fs')
+
+const COLORS = {
+  RESET: '\x1b[0m',
+  BLUE: '\x1b[34m',
+  GREEN: '\x1b[32m',
+  RED: '\x1b[31m',
+  YELLOW: '\x1b[33m',
+}
```

Comment:
There're several reasons:
- We have to introduce new 3rd party libraries in `dependencies` instead of `devDependencies`, which can lead to further work in maintenance.
- There're only a few colours to show, then we don't need such a big solution.

Reply:
Ah, yes. That's true that it would need to be moved to `dependencies` for this script where the release one is internal use only.